includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -3751,7 +3840,12 @@ class AMP_Allowed_Tags_Generated {
 					),
 					'credentials' => array(),
 					'data-amp-bind-is-layout-container' => array(),
-					'data-amp-bind-src' => array(),
+					'data-amp-bind-src' => array(
+						'mandatory_anyof' => array(
+							'src',
+							'[src]',
```

Comment:
This should get generated as `'data-amp-bind-src'`. See https://github.com/ampproject/amp-wp/pull/4285/files#r378581454.

Reply:
Good idea, it's applied in https://github.com/ampproject/amp-wp/compare/39926b690632afdfe86458844d5e34614e85fb76...bcec1fa7e40d487b656f5dde699c650ef72a06cc